src/dataSource.ts

```diff
@@ -11,13 +13,15 @@ if (process.env.NODE_ENV === 'test') {
 } else {
   dataSource = new DataSource({
     type: 'postgres',
-    host: Config.database.host,
-    port: Config.database.port,
-    username: Config.database.username,
-    password: Config.database.password,
-    database: Config.database.name,
-    synchronize: false,
-    logging: false,
+  host: Config.database.host,
+  port: Config.database.port,
+  username: Config.database.username,
+  password: Config.database.password,
+  database: Config.database.name,
+  entities: [Coin, CoinPrice], 
+  migrations: [ CreateCoinAndCoinPriceTables1631234567894 ],
+  synchronize: false,
+  logging: false,
```

Comment:
Good WORK!
You've remembered to set the logging to false :)

Reply:
:))))